modules/flowable-engine/src/test/java/org/flowable/standalone/testing/FlowableTestTaskInQueryTest.java

```diff
@@ -0,0 +1,51 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.flowable.standalone.testing;
+
+import org.flowable.engine.test.Deployment;
+import org.flowable.engine.test.FlowableTestCase;
+import org.flowable.task.api.Task;
+import org.flowable.task.api.history.HistoricTaskInstance;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Thanks for the PR.

Please follow project coding standards for imports; see https://github.com/flowable/flowable-engine/tree/main/ide-settings.  

The standard order is:

- static imports
- java
- javax
- org
- com
- everything else